### PR TITLE
Adding hover state for profile widget

### PR DESCRIPTION
**Pull request by adamjgrant**
@cvializ

### CODE REVIEW

File: kifi-backend/modules/shoebox/angular/src/profile/profileWidget.styl [hunk 1]
```diff
@@ -78,15 +78,14 @@
...
-    .stat
-      width: 33%
-
+    tr
       &:hover
-        h2
-          text-decoration: underline
+        td:last-child
+          .stat
+            text-decoration: underline
```
Comment (cvializ) [inline kifi-backend/modules/shoebox/angular/src/profile/profileWidget.styl:85]: I think long-term that giving class names to the `td` elements is the most maintainable way to style these elements, but this works

**Comment by cvializ** [comment]
:+1:

File: kifi-backend/modules/shoebox/angular/src/profile/profileWidget.styl (hunk 1 above)
Comment (adamjgrant) [inline kifi-backend/modules/shoebox/angular/src/profile/profileWidget.styl:85]: It's probably a better strategy for consistency of this application. I think outside of it, rarer class names enforces better namespacing and leads to cleaner markup.

Comment (adamjgrant) [inline kifi-backend/modules/shoebox/angular/src/profile/profileWidget.styl:85]: For the first point, I'll take another look at it and add some classes

**Pull request merged**